### PR TITLE
add postgresql meta db table schema configuration property (#7137)

### PR DESCRIPTION
**Pull request by Jinstarry**
If the postgresql db schema changes, you must set the configuration
values.
You do not need to set it if there is no change from the default schema
'public'.
druid.metadata.postgres.dbTableSchema=public

### CODE REVIEW

File: extensions-core/postgresql-metadata-storage/src/main/java/org/apache/druid/metadata/storage/postgresql/PostgreSQLMetadataStorageModule.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public void configure(Binder binder)
   {
     super.configure(binder);
 
-    JsonConfigProvider.bind(binder, "druid.metadata.postgres.ssl", PostgreSQLConnectorConfig.class);
+    JsonConfigProvider.bind(binder, "druid.metadata.postgres", PostgreSQLConnectorConfig.class);
```
Comment (drcrallen) [inline extensions-core/postgresql-metadata-storage/src/main/java/org/apache/druid/metadata/storage/postgresql/PostgreSQLMetadataStorageModule.java]: is this change intended?

**Comment by Jinstarry** [comment]
> Thanks for the contrib!
> 
> `PostgreSQLConnectorConfig` should be centered around ssl, and if there are non ssl configs, it probably needs to be its own class. That will avoid suddenly breaking ssl configs in deployments

Hi, 
Oh, I thought the config class name was a very common name. So I used 'PostgreSQLConnectorConfig' to set 'dbTableSchema'.
If so, I will create a 'PostgreSQLTablesConfig' class to set   'dbTableSchema'.

**Comment by jihoonson** [comment]
@Jinstarry the latest change looks good to go. Would you please check @drcrallen's last comment?
